src/components/collection_vote_box/CollectionVoteButton.js

```diff
@@ -0,0 +1,52 @@
+import {useEffect, useState} from "react";
+import {button} from "react-hyperscript-helpers";
+
+const styles = {
+  baseStyle: {
+    height: '45px',
+    width: '94px',
+    borderRadius: '5px',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    fontSize: '18px',
+    padding: '5%'
+  },
+  defaultBackgroundColor: '#FFFFFF',
+  defaultLabelColor: '#333F52',
+  selectedLabelColor: '#FFFFFF'
+};
+
+export default function CollectionVoteButton(props) {
+  const [additionalStyle, setAdditionalStyle] = useState({});
+  const { onClick, label, disabled, isSelected, baseColor, dataCy } = props;
+
+  useEffect(() => {
+    isSelected ? selectedButtonStyle() : defaultButtonStyle();
+  }, [isSelected]);
+
+  const defaultButtonStyle = () => {
+    updateStyle(styles.defaultBackgroundColor, styles.defaultLabelColor, false, disabled);
```

Comment:
> The reason why the linter is throwing a warning is that this function, alongside selectedButtonStyle, use prop values to execute a function. Since prop values can change on a parent update, the functions' definition needs to be updated as well, so it'll have to be redefined on prop update.

FYI, I'm not seeing any linter errors for this - let's see if we can figure out why different folks are or are not seeing what you're seeing @JVThomas